### ISSUE TITLE
Question about the parameter of normalization at the input.( mean=[0.485, 0.456, 0.406],   std=[0.229, 0.224, 0.225] ))

### ISSUE THREAD

**Issue by RaymondLmx** [Nanne/pytorch-NetVlad#33]
```
def input_transform():
    return transforms.Compose([
        transforms.ToTensor(),
        transforms.Normalize(mean=[0.485, 0.456, 0.406],
                               std=[0.229, 0.224, 0.225]),
    ])
```

I find the normalization in this transform is same between both Pittsburge and Tokyo dataset. Why different datasets can use same normalize parameter here? 

If I use grey-scale image (transform it to 3-channel RGB image by copying 1 channel) here, how should I choose or calculate normalize parameter here?

**Comment by Nanne** [comment]
The normalisation isn't tuned to the dataset, instead we used the normalization that was used to train the backbone network. Since those were trained on ImageNet, this normalisation matches that: https://github.com/pytorch/examples/blob/97304e232807082c2e7b54c597615dc0ad8f6173/imagenet/main.py#L197-L198

If your input is that different then the backbone network might not work very well, which means the selection of positives and negatives doesnt work well, which could explain why your training loss isnt decreasing. 

It sounds like you're trying to do something custom, and while I would love to help, I simply don't have the time, I would recommend to just play around with things and see if you can get any results.
